src/batch-client/index.ts

```diff
@@ -916,19 +884,23 @@ export class ZimbraBatchClient {
 			message: messages && messages.map(this.normalizeMessage)
 		}));
 
-	public search = (options: SearchOptions) =>
-		this.jsonRequest({
+	public search = (options: SearchOptions) => {
+		return this.jsonRequest({
```

Comment:
not sure why we needed to add return statement here, it was implicit return earlier

Reply:
I had initially added the logic to change the other attribute in the body as per expected by api, I have now moved it to a separate file but missed to remove return statement.
Removed it.